bitnami/clickhouse/values.yaml

```diff
@@ -1057,8 +1455,9 @@ serviceAccount:
   ## @param serviceAccount.automountServiceAccountToken Automount service account token for the server service account
   ##
   automountServiceAccountToken: false
-## Prometheus metrics
-##
+
+## @section Prometheus metrics parameters
+
 metrics:
   ## @param metrics.enabled Enable the export of Prometheus metrics
```

Comment:
Now that we're doing a major, could you remove the templated part in podAnnotations?

Reply:
done at https://github.com/bitnami/charts/pull/32958/commits/786dc3e86a09575f9fb564f1362f6c4b56c4e837